source/docs/software/examples-tutorials/third-party-examples.rst

```diff
@@ -0,0 +1,11 @@
+Third Party Example Projects
+============================
+
+This list helps you find example programs for use with third party devices.  You can find support for many of these third parties on the :ref:`docs/software/support/support-resources:Support Resources` page.
+
+* `Cross The Road Electronics (CTRE) <https://github.com/CrossTheRoadElec/Phoenix-Examples-Languages>`__
+* `REV Robotics <https://github.com/REVrobotics/SPARK-MAX-Examples>`__
+* `Limelight <https://docs.limelightvision.io/en/latest/cs_drive_to_goal_2019.html>`__ (more case studies can be found on the left)
```

Comment:
```suggestion
* `Limelight <https://docs.limelightvision.io/en/latest/cs_drive_to_goal_2019.html>`__ (more case studies can be found on the left)
```

As a reader, I don't know what "more" is really referring to as there is no noun that it was previously describing. Also the reader has no idea what a "case study" is. I'd either omit this entire sentence or rephrase to something close to "example code can be found in the left menu in the form of case studies", but that is quite lengthy.